addons/web/static/src/legacy/scss/kanban_view.scss

```diff
@@ -521,7 +516,7 @@
                 }
 
                 .o_column_unfold {
```

Comment:
is it not possible to add `d-block` when `o_column_unfold` is toggled directly?

Reply:
Here it’s in legacy, but it’s the case in the new component : 
https://github.com/odoo/odoo/pull/96195/files#diff-823da6a5974aafef6ec7d54b4d410ccd8dc54e1892aee57c3bb62ff8a0414826R54